src/compiler/checker.ts

```diff
@@ -14308,6 +14310,10 @@ namespace ts {
             return (deferredGlobalBigIntType ||= getGlobalType("BigInt" as __String, /*arity*/ 0, /*reportErrors*/ false)) || emptyObjectType;
         }
 
+        function getGlobalRecordSymbol() {
+            return deferredGlobalRecordSymbol ||= getGlobalTypeAliasSymbol("Record" as __String, /*arity*/ 2, /*reportErrors*/ true) || unknownSymbol;

```

Comment:
`unknownSymbol` is declared with `SymbolFlags.Property` - is this the right thing to use?

Reply:
Maybe just make this possibly return `undefined`

Reply:
Elsewhere we have assumptions about `unknownSymbol` being `SymbolFlags.Property`. Plus it has to be _something_.